src/site/content/en/blog/native-lazy-loading/index.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Native lazy loading for the web
+subhead: Browser-level native lazy loading is finally here!
+authors:
+  - houssein
+  - addyosmani
+  - mathiasbynens
+date: 2019-06-20
+hero: hero.png
+alt: Phone outline with loading image and assets
+description: |
+  This post covers the loading attribute and how it can be used
+  to defer the loading of images and iframes.
```

Comment:
> defer

Probably not important (as the objective of the article may be to motivate developers to **defer** loading of images/frames), but It'd be more technically correct to say "control" here, as the attribute can be used to override `loading-frame-default-eager 'none'`, using `loading="eager"`.